### DISCUSSION TITLE
Move official IRC channel from freenode to Libera.Chat

### DISCUSSION THREAD

**Discussion by Freso** [beetbox/beets#3953] [Ideas]
freenode has had a hostile takeover (see [DWF CVE-2021-1000189](https://github.com/distributedweaknessfiling/dwflist/blob/main/2021/1000xxx/CVE-2021-1000189.json)).

I have personally registered `#beets` on [Libera.Chat](https://libera.chat/). [`#whipper` has already moved over](https://github.com/whipper-team/whipper/issues/540), I expect MetaBrainz will move over sometime after our Monday meeting the day after tomorrow (any interested parties are welcome in joining for that meeting).

**Comment by wisp3rwind** [comment]
In principle, sounds good to me. I'm still confused about what is really going on: I've read some of the resignation letters and logs, and people seem to be blaming each other a lot, but ultimately there appears to be very little actual information about what happened in the background. As mostly an outsider to IRC and freenode in particular, I can't really judge whether or not put trust in either the new future of freenode or libera.chat. Certainly, the organizational structure of freenode must be problematic. libera.chat apparently is moving to change that: https://libera.chat/bylaws

In any case, thanks for reserving `#beets` and bringing up the topic.

**Reply by Freso** [reply]
> people seem to be blaming each other a lot

They do? I see rasengan blaming freenode staff, and freenode staff blaming rasengan. I’m obviously biased being part of the freenode staff that resigned (and as of yesterday part of Libera.Chat’s staff and today a member of the non-profit), but I haven’t seen other fingerpointing than this.

And, whether you trust me or not is of course up to you, but FWIW, my experience and recollection of events align with the ones other staffers have outlined in their resignation letters, and not with Mr. Lee’s.

**Reply by wisp3rwind** [reply]
> rasengan blaming freenode staff, and freenode staff blaming rasengan

is what I was talking about. Generally, your view of the events seems more plausible to me, too. As a bystander, I found it rather difficult to form a coherent picture of what's going on, though. Let's wait for a few other opinions here, but I tend to agree to switch.

> And, whether you trust me or not is of course up to you, but FWIW, my experience and recollection of events align with the ones other staffers have outlined in their resignation letters, and not with Mr. Lee’s.

Good to know! As I said, I've not been around on any channel on freenode (or any other network for that matter) much, so none of the names I've read had a familiar ring up to now :)

**Comment by sampsyo** [comment]
Thanks for the heads up, @Freso, and for registering the new channel. I suppose the next step would be to change our docs and website to point to the new place (which I'm happy to do)?

**Reply by wisp3rwind** [reply]
I guess the topic in `freenode/#beets` should also be changed to point at the new channel.

**Reply by Freso** [reply]
I could only find one mention of freenode in our codebase: https://github.com/search?q=org%3Abeetbox+freenode&type=code

I made a PR for changing that: https://github.com/beetbox/beets/pull/3957

I can handle freenode topic and locking down the channel if wanted.

**Reply by Freso** [reply]
I’ve locked (`+m`) the freenode channel now and put in a topic redirecting to Libera.Chat (and this discussion).
